lib/bindings.js

```diff
@@ -1 +1,5 @@
-module.exports = require('bindings')('xmljs');
+var binary = require('node-pre-gyp');
+var path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname, '../package.json')));
```

Comment:
to keep using bindings, this would also work

```module.exports = require('bindings')({
  bindings: 'xmljs',
  try: [['module_root', 'lib', 'binding', 'node-v' + process.versions.modules + '-' + process.platform + '-' + process.arch, 'bindings']]
});```